.github/helper/install_dependencies.sh

```diff
@@ -2,6 +2,13 @@
 
 set -e
 
+# Check for merge conflicts before proceeding
+python -m compileall -f "${GITHUB_WORKSPACE}"
+if grep -lr --exclude-dir=node_modules "^<<<<<<<" "${GITHUB_WORKSPACE}"
+    then echo "Found merge conflicts"
+    exit 1
```

Comment:
Possible false positives. Very rare, but possible xD

Reply:
Doubt anywhere in code we'd have code starting with `<<<<<<`? regex considers from start of line 🤔 

thought of adding word boundary at the end but then multiple flavors of grep regex bamboozled me so reverted it.